cockatrice/src/logger.cpp

```diff
@@ -51,16 +51,18 @@ void Logger::closeLogfileSession()
 void Logger::log(QtMsgType /* type */, const QMessageLogContext & /* ctx */, const QString &message)
 {
     logBuffer.append(message);
-    if(logBuffer.size() > LOGGER_MAX_ENTRIES)
-        logBuffer.removeFirst();
+    if (logBuffer.size() > LOGGER_MAX_ENTRIES)
```

Comment:
Do we want size or length here? Does logBuffer have both? I think for Qt size usually refers to byte size and length is number of elements. Maybe that's the source of the problem?